FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Models/FhirApiOption/BaseSearchOptions.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Health.Fhir.Synapse.DataClient.Models.FhirApiOption
 {
-    public class BaseSearchOptions : BaseFhirApiOptions
+    public class BaseSearchOptions : BaseApiOptions
```

Comment:
Add `Fhir` prefix to the `BaseSearchOptions`, and same for other options.

Reply:
I think it would be OK as they are placed under different namespaces. This one is placed under `FhirApiOption` while the ones for DICOM are placed under `DicomApiOption`.